src/interpreter/stepper.rs

```diff
@@ -151,9 +151,10 @@ impl<'a, 'b, 'mir, 'tcx> Visitor<'tcx> for ConstantExtractor<'a, 'b, 'mir, 'tcx>
             // already computed by rustc
             mir::Literal::Value { .. } => {}
             mir::Literal::Item { def_id, substs } => {
-                let item_ty = self.gecx.tcx.lookup_item_type(def_id).subst(self.gecx.tcx, substs);
```

Comment:
I think this change also applies to https://github.com/solson/miri/pull/18/files#diff-b6ba58183c703f8ffcaa54879146b1c2R1192